src/components/document-metadata-modal.yml

```diff
@@ -22,9 +22,21 @@ invalidTags:
 tagsPlaceholder:
   en: Enter keywords
   de: Stichwörter angeben
-useTemplateDocument:
-  en: Use default template
-  de: Standardvorlage benutzen
+tagsInfo:
+  en: Tags make the document discoverable when searching by words that match them
+  de: Tags make the document discoverable when searching by words that match them [but in German]
+content:
+  en: Content
+  de: Inhalt
+contentEmpty:
+  en: empty
+  de: Lehr
```

Comment:
It would be "Leer" - but what about:

Kein Inhalt - No content
Inhalt aus Vorlage - Content from template

?

Reply:
Since the label is already "Content" / "Inhalt", it would sound repetitive to put the word also in each radio option.
How does this look?
<img width="203" alt="Screen Shot 2023-01-26 at 12 01 54" src="https://user-images.githubusercontent.com/8189923/214820163-4b5b9201-b2cf-4469-9b1b-6d4cdd00f5ed.png">
<img width="236" alt="Screen Shot 2023-01-26 at 12 02 07" src="https://user-images.githubusercontent.com/8189923/214820168-dcb24dd8-ac15-4943-b78e-f570de7894ce.png">

Maybe "kein", with small letters?

Reply:
I would always capitalize the first word of each control. Unfortunately "Kein(er)" cannot stand alone in German, it just sounds weird, while in English "none" alone does work. I can only think of the two options:

* Repeating the word "Inhalt" as in "Kein Inhalt"
* Reverting back to "Leer" (but "Leerer Inhalt"/"Empty content" is as geeky as "empty string" or "empty set", for "normal" people, "Inhalt"/"Content" already excludes emptiness)

Reply:
Looking at other cases, we went for "Kein <Nomen>" so let's then go with:
- [en] "none" / "from template" (here we can afford to drop the repetition)
- [de] "Kein Inhalt" / "Inhalt aus Vorlage" (here if we repeat in one it sounds off to not repeat in the other)